hologram_dashboard/extension/src/app/dashboard/dashboard.component.ts

```diff
@@ -1,19 +1,23 @@
 import { Component, OnInit } from '@angular/core';
-import { RequestHandler } from '../request-handler.component'
+import { RequestHandler } from '../request-handler.service';
 
-// Interface for keeping track of whether a data set is available for a specific
-// date.
+/**Interface for keeping track of whether a data set is available for a 
+ * specific date. 
+ */
 export interface DateAvailability {
   date: string;
   isAvailable: boolean;
 }
 
-// Interface for keeping track of all the availability of a specific source.
+/**Interface for keeping track of all the availability of a specific source. */
 export interface HologramDataAvailability {
   sourceType: string;
-  dateAvailabilityList: Array<DateAvailability>
+  dateAvailabilityList: Array<DateAvailability>;
 }
 
+/**Component for using the retrieved protobuf from database and rendering it
```

Comment:
actually the retrieving from DB is not part of this component. It is to render the data availability dashboard.